src/scripts/asound.template.conf

```diff
@@ -0,0 +1,21 @@
+pcm.!default {
+	type hw
+	card 0
+}
+
+ctl.!default {
+	type hw
+	card 0
+}
+
+
+pcm.shairportdev {
```

Comment:
Can you maybe put this stuff into an extra config file so we don't have to rewrite the main ALSA config? I'm not sure what's the best approach here.